test/test_ad4020.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+hardware = "ad4020"
+classname = "adi.ad4020"
+
+#########################################
+@pytest.mark.iio_hardware(hardware)
```

Comment:
Should be @pytest.mark.iio_hardware(hardware, True) otherwise [emulation support will turn on](https://pytest-libiio.readthedocs.io/en/latest/emulation/)